src/mpyl/validation.py

```diff
@@ -60,6 +60,12 @@ def allow_none_validator(validator, types, instance, yaml_schema):
     return extended_validator(schema=schema, registry=registry)
 
 
+def validate_run_properties(properties: dict):
+    build_dict = pkgutil.get_data(__name__, "schema/run_properties.schema.yml")
```

Comment:
```suggestion
    schema = pkgutil.get_data(__name__, "schema/run_properties.schema.yml")
```
maybe? since we're in the context of `validate_run_properties` we don't need to call it `run_properties_schema` I think